nostrudel/umbrel-app.yml

```diff
@@ -0,0 +1,42 @@
+manifestVersion: 1
+id: nostrudel
+category: social
+name: noStrudel
+version: "0.37.1"
+tagline: A sandbox for exploring nostr
+icon: https://raw.githubusercontent.com/hzrd149/umbrel-community-app-store/master/hzrd149-nostrudel/icon.svg
+description: >-
+  noStrudel is a nostr web client that is focused on exploring the nostr network.
+
+  The focus is on power users or any other users who want to learn the inner workings of nostr.
+releaseNotes: >-
+  Changes:
+
+   - Rebuild DMs view to be more readable
+
+   - Show reactions and zaps on DMs
+
+   - Add support for threads in DMs
+
+   - Add support for Amber signer
+
+   - Add support for NIP-46 signer
+
+   - Support NIP-31 on unknown event kinds
+
```

Comment:
For this initial launch you can just use quotes here so that no release notes are shown:

```yaml
releaseNotes: ""
```

Reply:
updated :+1: